src/password/Create.vue

```diff
@@ -162,6 +168,10 @@ const maxLength = (v, vm) =>
 const strong = (v, vm) =>
   vm.strength.score >= vm.$root.idpConfig.passwordRules.minScore ||
   vm.$vuetify.lang.t('$vuetify.password.create.tooWeak')
+const requireAlphaAndNumeric = (v, vm) =>
+  !vm.$root.idpConfig.passwordRules.requireAlphaAndNumeric ||
+  (/\p{L}/u.test(vm.password) && /\p{N}/u.test(vm.password)) ||
+  vm.$vuetify.lang.t('$vuetify.password.create.requireAlphaAndNumeric')
```

Comment:
There is a log going on in this arrow function. For readability/maintainability, it may be worth breaking this out into slightly more readable code. Food for thought.